spec/api/bundes-api.spec.js

```diff
@@ -240,4 +278,60 @@ describe('Bundles API', () => {
                 .expect(400);
         });
     });
+
+
+    describe('put', () => {
+
+        it('should allow to update bundle by id', () => {
+            apiRequest
+                .put('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                .send({
+                    published: 1
+                })
+                .expect(200)
+                .then(() => {
+                    return apiRequest
+                        .get('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                        .expect(res => {
+                            expect(res.body).to.have.property('published')
+                                .and.equal(1);
+                        });
+                });
+        });
+
+        it('should reject update without any data', () => {
+
+            return apiRequest
+                .put('/v1/bundles/79dbdd38-1233-42f8-9d32-a1dad5871b52')
+                .send({})
+                .expect(500);
```

Comment:
I think for params related errors response code 40x works better than 50x since its request, not server issue. Is there any reason to have 500 here?